bylaws/2-cf-membership.md

```diff
@@ -30,13 +30,20 @@ A **project** within the CF represents collaborative innovation in open-source t
 
 **Dedicated Representation:** If an EGC representative is also elected to the [CF Council (CFC)][cfc], we recommend appointing an additional representative for the EGC. This helps maintain a focused approach to both specific project needs and the broader objectives of the CF.
 
+> [!NOTE]
+> An EGC member does not have to be a general CF member.
+> They are appointed by a project to represent its interests in [Foundation governance][governance].
+
 ## General Members
 
 **Join as an individual.** Membership to the CF is open to all individuals who share our mission and meet the outlined eligibility criteria[^3].
 
-- **Eligibility Criteria**: Eligibility requires active and meaningful engagement with CF or its projects for at least three months. Meaningful contributions include commits and merged pull requests (code, documentation, or applications for demonstration, education, or advocacy), or triage, maintainer or moderator roles for CF project repositories.
-- **Application Process:** Submit an application to formalize membership (*details TBD*). The CFC reviews applications to ensure eligibility criteria are met.
-- **Transparent Review Process:** Applicants satisfying criteria are accepted. Declinations are clearly explained based on set criteria, with an appeal process available for further evaluation.
+- **Eligibility Criteria**: Eligibility requires active and meaningful engagement with CF or its projects for at least three months.
+    Meaningful contributions include commits and merged pull requests (code, documentation, or applications for demonstration, education, or advocacy), or triage, maintainer or moderator roles for CF project repositories.
+- **Application Process:** Submit an application to formalize membership (*details TBD*).
```

Comment:
I wonder is this list should explicitly call out "user support" (i.e. answering questions) as a meaningful contribution.

Reply:
how about that?

Reply:
Yeah. Do you _agree_ that this can be a "meaningful contribution"?